cmd/kube-apiserver/app/server.go

```diff
@@ -416,7 +416,7 @@ func CreateKubeAPIServerConfig(
 		pubKeys = append(pubKeys, keys...)
 	}
 	// Plumb the required metadata through ExtraConfig.
-	config.ExtraConfig.ServiceAccountIssuerURL = s.Authentication.ServiceAccounts.Issuer
+	config.ExtraConfig.ServiceAccountIssuerURL = s.Authentication.ServiceAccounts.Issuers[0]
```

Comment:
are we guaranteed len(s.Authentication.ServiceAccounts.Issuers) > 0 here so this doesn't panic?

Reply:
it is guaranteed as long as the validation called. in the integration tests, the test server doesn't provide those options.

done.